_targets.R

```diff
@@ -1,4 +1,5 @@
 library(targets)
+library(tarchetypes)
```

Comment:
Since you are no longer using `tar_map()`, you actually shouldn't need this package.